### ISSUE TITLE
Nanogui invisible by default

### ISSUE THREAD

**Issue by kalhartt** [libigl/libigl#278]
The Nanogui panel isn't appearing in the examples with LIBIGL_WITH_NANOGUI ON.
I believe `screen->setVisible(true)` is missing from the end of Viewer::init [here](https://github.com/libigl/libigl/blob/master/include/igl/viewer/Viewer.cpp#L263)

**Comment by danielepanozzo** [comment]
I cannot reproduce the problem. Which tutorial has this problem and on which OS?

If adding that line fixes it for you, please send a pull request with the change. 

Thanks

**Comment by kalhartt** [comment]
I'm building on CentOS 7.2 with gcc 4.8.5 This happens on every tutorial, none of them have the gui visible for me. When I start one of the examples, I get the following output first:

```
igl_with_nanogui_defined_consistently: 1
igl_with_nanogui_defined_at_compile: 1
igl_with_nanogui_defined_at_include: 1
igl::viewer::Viewer usage:
...
```

So I think the GUI should be on. I'll attach a PR later today.

**Comment by decrispell** [comment]
Just want to add that I also am experiencing this issue with all tutorials (building on Linux Mint 17.3).  The gui shows up randomly once in a while, but usually not.  Perhaps the visible state is tied to some uninitialized memory?  In any case, adding the setVisible(true) line solves it.

**Comment by danielepanozzo** [comment]
Thank you for the report, it is fixed in 63bfc2d
